schemas/user.py

```
@@ -0,0 +1,97 @@
+        # 비밀번호를 입력하지 않은 경우(None)는 검증 통과 (수정 안 함)
+        if v is None:
+            return v
+
+        if not re.search(r"[a-zA-Z]", v):
+            raise ValueError("비밀번호에는 최소 하나 이상의 영문자가 포함되어야 합니다.")
+
+        if not re.search(r"\d", v):
+            raise ValueError("비밀번호에는 최소 하나 이상의 숫자가 포함되어야 합니다.")
+        return v
+
+    def check_password_update_requirements(self,v):
```

Ruff (0.14.11)

Unused method argument: `v`

(ARG002)